toast-tk-runtime/src/main/java/io/toast/tk/runtime/AbstractProjectRunner.java

```diff
@@ -152,11 +178,111 @@ protected void createAndOpenReport(final ITestPlan project) {
 			for (final ITestPage testPage : campaign.getTestCases()) {
 				String testPageHtmlReport = htmlReportGenerator.generatePageHtml(testPage);
 				htmlReportGenerator.writeFile(testPageHtmlReport, testPage.getName(), path);
+				System.out.println("\nNumber of Success step in test : " + testPage.getTestSuccessNumber() + "\n");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.talanlabs.com/coding_rules#rule_key=squid%3AS106)